Point-of-No-Return/Mapchip.cpp

```diff
@@ -53,11 +55,21 @@ void Mapchip::InitMap()
 		}
 	}
 	
-	for (int i = 0; i < world_size_height; i++)
+	for (int i = 0; i < MapChipInfo::world_size_height; i++)
 	{
 		map_[i] = map[i];
 	}
 }
 
 
 
+int Mapchip::CheckMapNumber(int x, int y, int** map)
+{
+	Matrix matrix = 
+	{
+		static_cast<Col>((x / MapChipInfo::chip_size) * MapChipInfo::chip_size),
```

Comment:
`(x / MapChipInfo::chip_size) * MapChipInfo::chip_size)` はintで、それを `Col` にstatic_cast？？
やるとしたら
```suggestion
		Col((x / MapChipInfo::chip_size) * MapChipInfo::chip_size),
```
じゃないですか？

Reply:
static_castじゃないですね。
修正しました。https://github.com/human-osaka-game-2019/Point-of-No-Return/pull/39/commits/8bc6f4515cf68f72672171a397d00def58bdc561